http-native/build.gradle

```diff
@@ -154,6 +154,7 @@ task extractJavaClassFiles(type: Copy) {
 }
 
 task codeCoverageReportJava(type: JacocoReport) {
+    dependsOn(':http-ballerina:build')
```

Comment:
This should depend on `:http-native:build`.

Reply:
yes, sorry i am in a hurry.